docs/file-reference.md

```diff
@@ -141,6 +143,7 @@ Each "app" (Kedge file) is a Kubernetes <a target="_blank" href="https://kuberne
 | ingresses | array of [ingress object](#ingressobject) | no           | [ingress object](#ingressobject) object |
 | routes | array of [route object](#routeobject) | no           | [route object](#routeobject)  object |
 | secrets | array of [secret](#secret) | no           | [secret](#secret) object |
+| imageStreams | array of [imageStream](#imageStream) | no           | [imageStream](#imageStream)  object |
```

Comment:
this should say ` array of [imageStreamObject](#imagestreamobject)` and last collum should be also imageSteamObject. Only first is imageStreams